docs/swarm/swarm-mode.md

```diff
@@ -73,10 +73,31 @@ To add a manager to this swarm, run the following command:
     192.168.99.100:2377
 ```
 
+### Configure the advertise address
+
+Manager nodes use an advertise address to allow other nodes in the swarm access
+to the API and overlay networking. The other nodes on the swarm must be able to
+access the manager node on its advertise address IP address.
+
+If you don't specify an advertise address, Docker checks if the system has a
+single IP address. If so, Docker uses the IP address with with the listening
+port `2377` by default. If the system has multiple IP addresses, you must
+specify the correct  `--advertise-addr` to enable inter-manager communication
+and overlay networking:
+
+```bash
+$ docker swarm init --advertise-addr <MANAGER-IP>
+```
+
+Refer to the `docker swarm init` [CLI reference](../reference/commandline/swarm_init.md)
+for more detail on the advertise address.
+
 ### View the join command or update a swarm join token
 
 The manager node requires a secret token for a new node to join the swarm. The
-token for worker nodes is different from the token for manager nodes.
+token for worker nodes is different from the token for manager nodes. Nodes only
+use the join-token once when they join the swarm. Rotating the join token after
+a node has already joined a swarm does not affect the node's swarm membership.
```

Comment:
`, but will ensure that the old token cannot be used by any new nodes attempting to join the swarm.`